### PR TITLE
Create FAQ

### PR DESCRIPTION
**Pull request by roryjamesallen**
At the suggestion of @gamecat69, create a frequently asked questions page to save people repeating each other in the Discord!

Very much a WIP but I'm opening this PR now so that anyone can suggest questions to add to it

### CODE REVIEW

File: frequently_asked_questions.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
+# FAQ
```
Comment (mjaskula) [inline frequently_asked_questions.md:1]: [required] Let's link to this doc from other places, such as the main readme and troubleshooting docs.

**Review by mjaskula — APPROVED** [review]
Looks good to me. I'd suggest merging this and adding new questions as they come up via new PRs.

File: frequently_asked_questions.md (hunk 1 above)
Comment (roryjamesallen) [reply frequently_asked_questions.md:1]: Sounds good to me

**Comment by roryjamesallen** [comment]
I've just added the section in troubleshooting for the module not reaching 10V as this was referred to in the FAQ, so makes sense to include in this PR

File: troubleshooting.md [hunk 2]
```diff
@@ -27,6 +27,32 @@ This means that your device is either not connected, or not being detected.
 4. Re-flash the firmware using the 'flash_nuke.uf2' found on the [Adafruit website](https://learn.adafruit.com/getting-started-with-raspberry-pi-pico-circuitpython/circuitpython), and then re-flash it again using the most recent firmware
 
 
+## Calibration gets stuck on 10V
+
+This means that the value the module recorded for 10V when you were sending in voltages to the analogue input canot be reached by the CV output.
```
Comment (Didah) [inline troubleshooting.md:32]:
```suggestion
This means that the value the module recorded for 10V when you were sending in voltages to the analogue input cannot be reached by the CV output.  
```

**Pull request merged**